app/dashboard/templates/shared/bounty-popover.html

```diff
@@ -44,6 +44,9 @@
         [[/for]]
       [[/if]]
     </div>
+    <div class="mt-2">
+      [[if expiredIcon === "true"]]<img src="{% static "v2/images/time-expired.svg" %}" class="expired-icon" />[[/if]][[:expired]]
```

Comment:
You can save one one property by checking just for `expired` 👍

Reply:
Thanks :)! That was indeed a better idea. Just refactored and removed the `expiredIcon` property.